lib/main.js

```diff
@@ -293,14 +293,16 @@ reporterPanel.sendAddonReports = function() {
                             by=220;
                         if (by<110)
                             by=110;
-                        let panelSize = 240+(by-110);
+                        let panelSize = 250+(by-110);
```

Comment:
Good catch; fixed this as well (and actually moved to a constant so it's easier to change in the future).